src/terraform/install.sh

```diff
@@ -26,9 +26,11 @@ TERRAFORM_DOCS_SHA256="${TERRAFORM_DOCS_SHA256:-"automatic"}"
 
 TERRAFORM_GPG_KEY="72D7468F"
 TFLINT_GPG_KEY_URI="https://raw.githubusercontent.com/terraform-linters/tflint/master/8CE69160EB3F2FE9.key"
-GPG_KEY_SERVERS="keyserver hkp://keyserver.ubuntu.com
+GPG_KEY_SERVERS="keyserver hkps://keyserver.ubuntu.com
 keyserver hkps://keys.openpgp.org
-keyserver hkp://keyserver.pgp.com"
+keyserver hkps://keyserver.pgp.com"
+GPG_OPTS=""
+if [ "${http_proxy}" != "" ]; then GPG_OPTS="--keyserver-options http-proxy=$http_proxy"; fi
```

Comment:
Can you create a [Feature option](https://github.com/devcontainers/features/blob/ea03d367f118b11b2a85149781dc16083d977e5e/src/terraform/devcontainer-feature.json#L7-L46) for this? (say `httpProxy`; default `""`)

Reply:
As you pointed out, GPG_OPTS is not used.  I will Delete.
Thank you.

Reply:
Removed GPG_OPTS as it is not used.

Reply:
I don't know how to test features in pull requests.

Can I test by setting "ghcr.io/devcontainers/features/terraform:branch-name"?

Reply:
Feel free to take a look at https://github.com/devcontainers/cli/blob/main/docs/features/test.md for guidance.

> if I add http_proxy to the features options, won't the environment prevent the features from starting?

Isn't `htppProxy` only needed for downloading gpg keys? By default the value of this option will be `""`, people who are blocked and needs a proxy can only pass a value to this Feature.

If I have misunderstood your question, do you mind explaining your thoughts behind how/why the environment might prevent the features from starting?

Reply:
Thanks for giving me the link for the guidance!
I'll take a look at the documentation.

Reply:
I would like to use the development container with the same code in both a proxy environment and an internet-facing environment.

Reply:
If http_proxy is set in features options, it will work in the proxy environment, but I think it will be an error in other environments.

I think that it is better to set only the environment variable and refer to it from the code instead of setting the proxy directly in the code.

Reply:
I understand your comment.
Added http_proxy to features option.

I ran the test with the following command and it passed.
thank you very much.

```
node ➜ /workspaces/features (terraform-gpg) $ devcontainer features test --features terraform --base-image ubuntu

  ================== TEST REPORT ==================
✅ Passed:      'terraform'
✅ Passed:      'install_sentinel'
✅ Passed:      'install_tfsec'
✅ Passed:      'install_terraform_docs'
```